File: demo-iys-rest-scraping.java/src/main/java/demo/Application.java
package demo;

import demo.reader.Reader;
import demo.reader.SkillsReader;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class Application {

    private static final Logger logger = LoggerFactory.getLogger(Application.class);

    public static void main(String[] args) throws Exception {
        logger.info("application started");

        Reader reader = new SkillsReader();
        reader.read();

        logger.info("application finished");
    }
}

File: demo-iys-rest-scraping.kotlin/src/main/kotlin/demo/Application.kt
package demo

import demo.reader.SkillsReader
import org.slf4j.LoggerFactory

object Application {

    private val logger = LoggerFactory.getLogger(Application::class.java)

    @Throws(Exception::class)
    @JvmStatic fun main(args: Array<String>) {
        logger.info("application started")

        val reader = SkillsReader()
        reader.read()

        logger.info("application finished")
    }
}
